internal/prompts/app_select_test.go

```diff
@@ -2455,2328 +1313,192 @@ func TestPrompt_AppSelectPrompt_FlatAppSelectPrompt(t *testing.T) {
-
-			// Expected result
-			SelectedApp{
-				App:  appTeam1Hosted,
-				Auth: authTeam1,
-			},
-			nil,
-
-			// Env + app install status
-			ShowHostedOnly,
-			ShowInstalledAppsOnly,
-
-			// flags
-			"",
-			"",
-
-			// Auth state
-			[]types.SlackAuth{
-				authTeam1,
-			},
-			nil,
-
-			// Selector preferences team 1
-			team1TeamDomain,
-			"Deployed",
-		},
-	}
-
-	for _, test := range tests {
-
-		// Set app flags
-		clients.Config.AppFlag = test.appFlag
-		clients.Config.TeamFlag = test.teamFlag
-
-		// Return the auth state depending on test specs
-		clientsMock.Auth.On(Auths, mock.Anything).Return([]types.SlackAuth{
-			authTeam1,
-		}, nil)
-
-		clientsMock.AddDefaultMocks()
-
-		// Finally we mock the select prompt based on test specs
-		clientsMock.IO.On(SelectPrompt, mock.Anything, appInstallPromptNew, mock.Anything, iostreams.MatchPromptConfig(iostreams.SelectPromptConfig{
-			Flag: clientsMock.Config.Flags.Lookup("team"),
-		})).Return(iostreams.SelectPromptResponse{
-			Prompt: true,
-			Option: test.selectTeamDomain,
-			Index:  0,
-		}, nil)
-		clientsMock.IO.On(SelectPrompt, mock.Anything, "Choose a deployed environment", mock.Anything, iostreams.MatchPromptConfig(iostreams.SelectPromptConfig{
-			Flag: clientsMock.Config.Flags.Lookup("team"),
-		})).Return(iostreams.SelectPromptResponse{
-			Prompt: true,
-			Option: test.selectEnvironment,
-			Index:  0,
-		}, nil)
-		clientsMock.IO.On(SelectPrompt, mock.Anything, "Choose a local environment", mock.Anything, iostreams.MatchPromptConfig(iostreams.SelectPromptConfig{
-			Flag: clientsMock.Config.Flags.Lookup("team"),
-		})).Return(iostreams.SelectPromptResponse{
-			Prompt: true,
-			Option: test.selectEnvironment,
-			Index:  0,
-		}, nil)
-		// App selector mock
-		clientsMock.IO.On(SelectPrompt, mock.Anything, "Choose an app environment", mock.Anything, iostreams.MatchPromptConfig(iostreams.SelectPromptConfig{
-			Flag: clientsMock.Config.Flags.Lookup("app"),
-		})).Return(iostreams.SelectPromptResponse{
-			Prompt: true,
-			Option: "deployed",
-			Index:  1,
-		}, nil)
-
-		actualSelected1, err := TeamAppSelectPrompt(ctx, clients, test.env, test.status)
-		actualSelected2, err2 := AppSelectPrompt(ctx, clients, test.status)
-
-		if test.expectedError == nil {
-			require.NoError(t, err)
-			require.NoError(t, err2)
-
-			// There's a valid auth for this auth record
-			_, err := clients.AppClient().GetDeployed(ctx, test.expected.Auth.TeamID)
-			require.NoError(t, err)
-
-			require.Equal(t, test.expected, actualSelected1)
-			require.Equal(t, test.expected, actualSelected2)
-		} else if assert.Error(t, err) {
-			assert.Equal(t, test.expectedError.Code, err.(*slackerror.Error).Code, test.desc)
-
-			require.Equal(t, test.expected, actualSelected1)
-			require.Equal(t, test.expected, actualSelected2)
-		}
-	}
-}
-
-func TestPrompt_TeamAppSelectPrompt_AppSelectPrompt_EnterpriseWorkspaceApps_MissingWorkspaceAuth_MissingOrgAuth_UserReAuthenticates(t *testing.T) {
```

Comment:
🗣️ Reauthentication tests for related logic are kept in:

https://github.com/slackapi/slack-cli/blob/f9b74e0086fcb202435e92edf8f572d357037e89/internal/prompts/app_select_test.go#L1648-L1671